README.md

```diff
@@ -34,6 +34,8 @@ Espoo-dev is a learning platform. There are three user types:
   - **Web Client**: The users (Student) can answer `Questions` that belongs to a `Survey`
 - frontend (Next, React):
   - **Web Client**: The users (Student) can answer `Questions` that belongs to a `Survey`
+- Mobile (Android, Kotlin):
+  - **Mobile Client**: The users (Student) can answer Questions that belongs to a Survey
```

Comment:
@philipeleandro , this file is a .md file, check the preview and you will see that this line looks different from the others.

Reply:
@edimossilva thank you for the view, it will be adjusted.